bin/lint

```diff
@@ -47,10 +47,25 @@ _bash() {
   print_green "  -> Bash lint OK"
 }
 
+_plugins() {
+  echo -e "\n---> Plugins: flake8\n"
+  _docker_run flake8 plugins
+  print_green "  -> Flake8 check OK"
+
+  echo -e "\n---> Plugins: isort\n"
+  _docker_run isort --recursive --diff --check-only plugins
+  print_green "  -> isort check OK"
+
+  echo -e "\n---> Plugins: Pylint\n"
+  _docker_run pylint plugins
+  print_green "  -> Pylint check OK"
+}
+
```

Comment:
no black?

Reply:
`black` requires python 3.6+. I've tried this upgrade since Ansible 2.5+ is compatible with python 3 but we are running `debian:stretch` which is limited to python 3.5 :( We can consider switching to an ubuntu-based image, but does it worth it?

Reply:
Ok I'll declare a new issue for this.

Reply:
Done in #78 & #79